release/ray_release/test.py

```diff
@@ -340,7 +341,7 @@ def get_anyscale_byod_image(self) -> str:
         )
 
     def get_test_results(
-        self, limit: int = 10, refresh: bool = False
+        self, limit: int = 30, refresh: bool = False
```

Comment:
need longer series for flaky state check

Reply:
this feels like a magic number to me now.. could you explain why?

or maybe the caller should send this in?

also, it seems that it is not doing pagination on listing, so this limit actually cannot be very large?

Reply:
it's because of the conditions like `CONTINUOUS_FAILURE_TO_FLAKY = 10`, and later on the condition of passing --> flaky, we need more data to validate

let me move the limit to the state machine; 30 is not large i hope